internal/converter/testdata/standard/test.proto

```diff
@@ -196,4 +196,11 @@ message AllTypes {
     AllTypes msg_option = 96;
     Enum enum_option = 97;
   }
+
+  // oneof with optional fields
```

Comment:
I actually don't understand why it does not show up in generated file 🤔

Reply:
You have to run `make test` in order for these protobuf files to be updated. It's a bit of a quirk with how testing with this project is set up.

But also, it's not valid for fields in a `oneof` block to be `optional` apparently:
```
internal/converter/testdata/standard/test.proto:202:5:syntax error: unexpected "optional"
internal/converter/testdata/standard/test.proto:203:5:syntax error: unexpected "optional"
internal/converter/testdata/standard/test.proto:204:5:syntax error: unexpected "optional"
```

Reply:
thx, I'm checking whether it resolves the issue for us, and if so I will cycle back to finish the PR

Reply:
ok, it works for us, going to update tests